front-end/src/components/Map/LocationSelectMap.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Wrapper } from "@googlemaps/react-wrapper";
+import { CircularProgress } from "@mui/material";
+import TableMap from "./TableMap";
+
+const render = () => {
```

Comment:
Can we add another issue to add support to check the react-wrapper `status` and display an error message if there was a problem? I think right now we'll just always display the `CircularProgress` component even if loading the api failed.